app.rb

```diff
@@ -140,7 +137,6 @@ def create_a_rental
 
     puts 'Rental created successfully!'
     choose_a_number
-    puts
   end
 
   def list_all_rentals_for_a_person
```

Comment:
- [ ] in the `create_a_rental` method you are printing to the console getting information from the user and saving the information this is not in line with the single responsibility rule, have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more information.